src/ccm/batch-processing/templates/config.yaml

```diff
@@ -41,3 +41,5 @@ data:
   DATA_PIPELINE_CONFIG_GCP_PROJECT_ID: {{ .Values.GCP_PROJECT_ID  | quote}}
   DATA_PIPELINE_CONFIG_GCS_BASE_PATH: "gs://awscustomerbillingdata-onprem"
   STACK_DRIVER_LOGGING_ENABLED: "false"
+  NEXT_GEN_MANAGER_SECRET: IC04LYMBf1lDP5oeY4hupxd4HJhLmN6azUku3xEbeE3SUx5G3ZYzhbiwVtK4i7AmqyU9OZkwB4v8E9qM
```

Comment:
Lets move them in batch-processing backend soon https://github.com/harness/harness-core/blob/develop/batch-processing/service/batch-processing-config.yml#L145